Message.php

```diff
@@ -291,6 +291,21 @@ public function addTo($email, $name = '', $type = 'to')
     }
 
     /**
+     * Set recipients
+     * Accepts an array of values like Message->addTo() function. This will override the $to array.
+     *
+     * @param array $recipients
+     *
+     * @return Message
+     */
+    public function setTo($recipients = array())
+    {
+        $this->to = $recipients;
```

Comment:
Why you do not use the ``addTo`` method here ?

If there is a change on the declaration of each element of the property ``$to``, the project who use the ``setTo`` will have failures.